cli/main.go

```diff
@@ -9,25 +9,30 @@ import (
 )
 
 func main() {
+
 	app := &cli.App{
 		Name:  "Vault Helper Tool CLI",
-		Usage: "Lets user read, update and list user information.",
+		Usage: "Lets user read, update and list user information. If entering interactive mode, enter q or to quit",
 		Commands: []*cli.Command{
 			{
 				Name:    "write",
 				Aliases: []string{"w"},
-				Usage:   "update user information by overwriting (provide 1 argument - name of json file)",
+				Usage:   "update user information by overwriting (provide 2 arguments - token, json file)",
 				Action: func(c *cli.Context) error {
-					updateUserInfo(c.Args().Get(0))
+					cArg0 := c.Args().Get(0)
+					cArg1 := c.Args().Get(1)
+					callUpdate(cArg0, cArg1)
```

Comment:
As mentioned above, instead of using `callUpdate()` here, call the validator, error if the validator finds errors, otherwise call `updateUserInfo()` next. This feedback applies to all of the commands.